sessions/Sources/App/Models/User.swift

```diff
@@ -45,8 +45,21 @@ final class User: Model, HBAuthenticatable {
     }
 }
 
+/// User type to pass around as authenticatable. This is required as Fluent
+/// model types are not Sendable
+struct LoggedInUser: HBAuthenticatable {
+    let id: UUID
+    let name: String
+
+    init(from user: User) throws {
+        guard let id = try? user.requireID() else { throw HBHTTPError(.unauthorized) }
```

Comment:
Any reason not to re-throw the error from `requireID()` here?

Reply:
If we throw the error from requireID the server will return a 500 as it doesn't know how to convert that into a usable error. Maybe it should return a 500 as user really should have an ID

Reply:
I went with your suggestion in the end